core/src/test/scala/integration/kafka/api/AuthorizerIntegrationTest.scala

```diff
@@ -911,20 +913,20 @@ class AuthorizerIntegrationTest extends BaseRequestTest {
     assertEquals(offset, offsetFetchResponse.responseData.get(tp).offset)
   }
 
-  @Test
+  @Test(expected = classOf[org.apache.kafka.common.errors.TimeoutException])
```

Comment:
Why are we changing the behavior of these tests? They are intended to verify ACL behavior. We should just keep the current implementation.